main/plugins/org.talend.designer.bigdata.di/src/main/java/org/talend/designer/bigdata/di/components/thbase/THBaseConfiguration.java

```diff
@@ -3,9 +3,9 @@
 import java.util.HashMap;
 import java.util.Map;
 
-public class THBaseConfiguration {
+public class THBaseConfiguration<T extends THbase> {
```

Comment:
```suggestion
public class THbaseConfiguration<T extends THbase> {
```
To be aligned with other components naming

Reply:
done